### DISCUSSION TITLE
Remove Header x-powered-by

### DISCUSSION THREAD

**Discussion by erryKinq** [nextcloud/all-in-one#3403] [Ideas] [answered]
when calling my nextcloud I noticed that the header x-powered-by is supplied, which shows the currently used PHP version. 

I consider this to be a security vulnerability, as this can indicate possible security vulnerabilities that are known for the respective PHP version. 

Is there a special reason that this header is sent along? I would ask to remove this header from future Nextcloud-Aio versions if it is not needed.

BR
erryKinq

**Comment by Zoey2936** [comment, marked as answer]
this is done automatically by apache
